### PR TITLE
Fix ytt failing test with version 0.42 of ytt

### PR DESCRIPTION
**Pull request by squeedee**
Fix ytt failing test with version 0.42 of ytt
- we currently ship with 0.42
- we need to work out what's actually wrong with 0.43
- includes version display step for debug in the future

### CODE REVIEW

File: .github/workflows/validation.yaml [hunk 1]
```diff
@@ -40,6 +40,12 @@ jobs:
 
       - name: setup carvel tooling binaries
         uses: vmware-tanzu/carvel-setup-action@v1
+        with:
+          only: ytt
+          ytt: v0.42.0 # Todo unlock as soon as 0.43 is working.
+
+      - run: |
```
Comment (emmjohnson) [inline .github/workflows/validation.yaml:46]: do you want to leave the version command in?

**Review by emmjohnson — APPROVED** [review]

File: .github/workflows/validation.yaml (hunk 1 above)
Comment (squeedee) [reply .github/workflows/validation.yaml:46]: I'll take it out as soon as 0.43 is working.

Comment (squeedee) [reply .github/workflows/validation.yaml:46]: (in a future pr)

**Comment by squeedee** [comment]
Ok well this is not passing all the time either. Looks like it's not the version.

**Pull request merged**